src/test/java/com/rackspace/ceres/app/web/QueryControllerTest.java

```diff
@@ -101,7 +103,7 @@ public void testQueryApiWithAggregator() {
     Flux<QueryResult> result = webTestClient.get()
         .uri(uriBuilder -> uriBuilder.path("/api/query")
             .queryParam("tenant", "t-1")
-            .queryParam("metricName", "cpu-idle")
+            .queryParam("metricKey", "cpu-idle")
```

Comment:
I dont see a test where you are just providing the metricGroup for querying against the api